### ISSUE TITLE
Chrome: Error in event handler for tabs.onUpdated: Error: Expected at most 0 arguments for show(), got 1

### ISSUE THREAD

**Issue by T-vK** [mozilla/webextension-polyfill#24]
![Screenshot](https://i.snag.gy/CaOIpV.jpg)

My extension works without any errors in the latest version of Firefox. In Chrome I get this error.
Honestly, I'm not sure what is causing this error. I am using `tabs.onUpdated` and within that I'm using `pageAction.show()`. If you need more information, tell me.

Edit:
Maybe the issue is that I'm calling browser.pageAction.show() with a tab id in as the first argument as required by Firefox. I guess Chrome doesn't support it? 
I tried to work around it using `browser.pageAction.show(browser.pageAction.show.length?tabId:undefined)`.   
But that didn't make any difference.

Edit2: 
It seems to be a misconfiguration in the browser-polifill library. Setting `minArgs` and `maxArgs` of `show` under pageAction to 1 seems to fix it.

Edit3:
Actually I'm still getting an error:  
`Error: Invocation of form pageAction.show(integer, function) doesn't match definition pageAction.show(integer tabId)`

Edit4:
Here is the code causing the error:
``` JavaScript
browser.tabs.query({}).then( tabs => {
    if (!tabs)
        return
    let promiseArray = []
    tabs.forEach(tab => {
        let tabId = tab.id
        promiseArray.push(browser.pageAction.show(tabId).then(()=>{
            return browser.pageAction.setIcon({tabId: tabId, path: 'disabled.ico'})
        }))
    })
    return Promise.all(promiseArray)
}).then(()=>{
    /* ... */
}).catch(console.error)
```

**Comment by Rob--W** [comment]
See #13 (and #9). For now, just use chrome.pageAction.setIcon (or apply PR #13).

**Comment by Rob--W** [comment]
Fixed by #59.
